### PR TITLE
Fixed new patient add and update in front end

### PR DESCRIPTION
**Pull request by rtemelkovski**
- You can now add/ update patient
- after you submit the format it will send you back to the previous page
- you will also reload the user list on new user creation
- titles for pages and button vary depending on whether you are editing or creating person

### CODE REVIEW

File: frontend/mobilityai_react/App/Lib/Api.js [hunk 1]
```diff
@@ -14,6 +14,20 @@ export function AddPatientData(patientData) {
         .catch((error) => console.log(JSON.stringify(error)));
 };
 
+export function updatePatientData(patientId, patientData) {
+    const route = `api/Patients/${patientId}?patientData=` + patientData;
```
Comment (teovoinea) [inline frontend/mobilityai_react/App/Lib/Api.js]: Nice 👍

File: frontend/mobilityai_react/App/Screens/NewPatientForm.js [hunk 2]
```diff
@@ -73,23 +82,53 @@ export default class PatientForm extends React.Component {
         // }
     }
 
+    loadStateFromApi(json){
+        const data = JSON.parse(json.data);
+        console.log(data)
+        this.setState({
```
Comment (teovoinea) [inline frontend/mobilityai_react/App/Screens/NewPatientForm.js]: Why doesn't `this.setState(data);` work?

**Comment by teovoinea** [comment]
Fixes #155, #140.

**Comment by teovoinea** [comment]
Add a button from the patient data page. to view this information.

File: frontend/mobilityai_react/App/Lib/Api.js (hunk 1 above)
Comment (pgunasekara) [reply frontend/mobilityai_react/App/Lib/Api.js]: 👌👌👌

**Pull request merged**